"use strict";(self.webpackChunklearngenomics_dev=self.webpackChunklearngenomics_dev||[]).push([[987],{3905:function(e,n,t){t.d(n,{Zo:function(){return u},kt:function(){return m}});var r=t(7294);function o(e,n,t){return n in e?Object.defineProperty(e,n,{value:t,enumerable:!0,configurable:!0,writable:!0}):e[n]=t,e}function a(e,n){var t=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);n&&(r=r.filter((function(n){return Object.getOwnPropertyDescriptor(e,n).enumerable}))),t.push.apply(t,r)}return t}function i(e){for(var n=1;n<arguments.length;n++){var t=null!=arguments[n]?arguments[n]:{};n%2?a(Object(t),!0).forEach((function(n){o(e,n,t[n])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(t)):a(Object(t)).forEach((function(n){Object.defineProperty(e,n,Object.getOwnPropertyDescriptor(t,n))}))}return e}function s(e,n){if(null==e)return{};var t,r,o=function(e,n){if(null==e)return{};var t,r,o={},a=Object.keys(e);for(r=0;r<a.length;r++)t=a[r],n.indexOf(t)>=0||(o[t]=e[t]);return o}(e,n);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);for(r=0;r<a.length;r++)t=a[r],n.indexOf(t)>=0||Object.prototype.propertyIsEnumerable.call(e,t)&&(o[t]=e[t])}return o}var l=r.createContext({}),c=function(e){var n=r.useContext(l),t=n;return e&&(t="function"==typeof e?e(n):i(i({},n),e)),t},u=function(e){var n=c(e.components);return r.createElement(l.Provider,{value:n},e.children)},p={inlineCode:"code",wrapper:function(e){var n=e.children;return r.createElement(r.Fragment,{},n)}},d=r.forwardRef((function(e,n){var t=e.components,o=e.mdxType,a=e.originalType,l=e.parentName,u=s(e,["components","mdxType","originalType","parentName"]),d=c(t),m=o,h=d["".concat(l,".").concat(m)]||d[m]||p[m]||a;return t?r.createElement(h,i(i({ref:n},u),{},{components:t})):r.createElement(h,i({ref:n},u))}));function m(e,n){var t=arguments,o=n&&n.mdxType;if("string"==typeof e||o){var a=t.length,i=new Array(a);i[0]=d;var s={};for(var l in n)hasOwnProperty.call(n,l)&&(s[l]=n[l]);s.originalType=e,s.mdxType="string"==typeof e?e:o,i[1]=s;for(var c=2;c<a;c++)i[c]=t[c];return r.createElement.apply(null,i)}return r.createElement.apply(null,t)}d.displayName="MDXCreateElement"},4890:function(e,n,t){t.r(n),t.d(n,{assets:function(){return u},contentTitle:function(){return l},default:function(){return m},frontMatter:function(){return s},metadata:function(){return c},toc:function(){return p}});var r=t(7462),o=t(3366),a=(t(7294),t(3905)),i=["components"],s={title:"Next-Generation Sequencing",track:"Next-Generation Sequencing"},l=void 0,c={unversionedId:"next-generation-sequencing/next-generation-sequencing",id:"next-generation-sequencing/next-generation-sequencing",title:"Next-Generation Sequencing",description:"Once collected, sequencing the sample begins. The sample is removed from",source:"@site/docs/03-next-generation-sequencing/02-next-generation-sequencing.md",sourceDirName:"03-next-generation-sequencing",slug:"/next-generation-sequencing/next-generation-sequencing",permalink:"/docs/next-generation-sequencing/next-generation-sequencing",draft:!1,editUrl:"https://github.com/stjude/learngenomics.dev/tree/main/docs/03-next-generation-sequencing/02-next-generation-sequencing.md",tags:[],version:"current",sidebarPosition:2,frontMatter:{title:"Next-Generation Sequencing",track:"Next-Generation Sequencing"},sidebar:"tutorialSidebar",previous:{title:"Sample Collection",permalink:"/docs/next-generation-sequencing/sample-collection"},next:{title:"Types of Sequencing",permalink:"/docs/next-generation-sequencing/types-of-sequencing"}},u={},p=[],d={toc:p};function m(e){var n=e.components,s=(0,o.Z)(e,i);return(0,a.kt)("wrapper",(0,r.Z)({},d,s,{components:n,mdxType:"MDXLayout"}),(0,a.kt)("p",null,'Once collected, sequencing the sample begins. The sample is removed from\nstorage, and a portion is taken to prepare what is called a "library" for\nsequencing. This is always performed in a wet lab environment, and the chemical\nmethods can be quite complex.'),(0,a.kt)("p",null,"While it sounds easy to read the entire genome by laying all of the chromosomes\nend-to-end, in practice, there are some problems with this approach. Chiefly,\nresearchers currently lack the biochemical processing abilities to extract and to\nread such long stretches of DNA quickly and accurately. Additionally, next\ngeneration sequencing technologies have their own set of limitations, including:"),(0,a.kt)("ul",null,(0,a.kt)("li",{parentName:"ul"},"Each nucleotide the sequencer calls is accompanied by a confidence score\nestimating how likely the recorded call is to be correct. In short, the\nsequencer's calls are of a probabilistic nature and do not have a zero error\nrate."),(0,a.kt)("li",{parentName:"ul"},"As the sequencer observes each nucleotide along the read, the confidence it\nhas in each new call drops so that if allowed to continue eventually the data\nquality of the last bases would be too low to use reliably."),(0,a.kt)("li",{parentName:"ul"},"Sequencers can only read DNA fragments because RNA is too unstable. So\nbiochemical methods are employed to reverse the transcription process and\nproduce a DNA fragment that captures the information in the RNA sequence.")),(0,a.kt)("p",null,"Given these physical limitations, this process, known as ",(0,a.kt)("strong",{parentName:"p"},"shotgun sequencing"),",\nlooks like the following:"),(0,a.kt)("ol",null,(0,a.kt)("li",{parentName:"ol"},"The sample's DNA is fragmented into billions of smaller DNA pieces that are\nsomewhere between tens to thousands of bases long."),(0,a.kt)("li",{parentName:"ol"},'This soup of DNA fragments with randomly distributed lengths is then put\nthrough a filtering step called "size selection". Only DNA fragments of a\ncertain length will remain (typically somewhere between 200-500 bp, where bp\nstands for base pair).'),(0,a.kt)("li",{parentName:"ol"},'Among many other important steps to prepare the DNA library for sequencing is\n"adapter ligation". This process involves tacking on special sequences to\nboth ends of the DNA fragment which is required by the sequencing process.'),(0,a.kt)("li",{parentName:"ol"},"Optionally, a chemical amplification step, known as Polymeraise Chain\nReaction (",(0,a.kt)("strong",{parentName:"li"},"PCR"),"), can be conducted to increase the amount of genomic\nmaterial available for sequencing."),(0,a.kt)("li",{parentName:"ol"},"The sample is loaded onto a flowcell and put into an Illumina sequencer,\nwhich is capable of reading millions of DNA fragments and capturing the\nresults digitally."),(0,a.kt)("li",{parentName:"ol"},'Once the sequencing is complete, the sequencer closes the sequencing session\nby creating a directory of BCL files (Illumina\'s proprietary storage\nformat). These are immediately converted to a FASTQ file (standard DNA\nstorage format), typically the "input" to a computational team.')),(0,a.kt)("p",null,"The figure below depicts how this process generates fragments of varying lengths\nand shows how each read is generated based on the fragments."),(0,a.kt)("p",null,(0,a.kt)("img",{alt:"Figure depicting multiple reads of varying lengths being sequenced.",src:t(5754).Z,width:"2569",height:"1911"})),(0,a.kt)("p",null,'For our paired-end sequencing example, it is important to note how the DNA\nsequencer reads the data in step 5. In each 200-500bp DNA fragment, the\nsequencer begins reading from one of the fragments and drops off somewhere in\nthe middle. It stores all of the nucleotides read from this end of the fragment\nwith their associated confidence scores as "Read 1". After this is complete, the\nsequencer reads from the opposite end of the fragment and stores the nucleotide\ncalls and confidence scores known as "Read 2". Together, the combination of a\nRead 1 from a DNA fragment and a Read 2 from the other end of the DNA fragment\nis called a "read pair" as denoted in the figure below.'),(0,a.kt)("p",null,(0,a.kt)("img",{alt:"Figure showing a read on a Flowcell zoomed in.",src:t(7814).Z,width:"2184",height:"768"})),(0,a.kt)("p",null,'After the steps above have completed, the result is a single FASTQ file (single-end sequencing) or a pair of FASTQ files (paired-end). For more information on the FASTQ file format, see its section in the "Genomic File Formats" chapter.'))}m.isMDXComponent=!0},5754:function(e,n,t){n.Z=t.p+"assets/images/3.1-Reads-on-Flowcell-df87b79255200e3d00ea94858f9616d6.jpg"},7814:function(e,n,t){n.Z=t.p+"assets/images/3.2-Read-on-Flowcell-Zoomed-dfd55a6034281db2d2e729bf64393b88.jpg"}}]);