"use strict";(self.webpackChunklearngenomics_dev=self.webpackChunklearngenomics_dev||[]).push([[94],{3905:function(e,n,t){t.d(n,{Zo:function(){return u},kt:function(){return m}});var i=t(7294);function a(e,n,t){return n in e?Object.defineProperty(e,n,{value:t,enumerable:!0,configurable:!0,writable:!0}):e[n]=t,e}function o(e,n){var t=Object.keys(e);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);n&&(i=i.filter((function(n){return Object.getOwnPropertyDescriptor(e,n).enumerable}))),t.push.apply(t,i)}return t}function r(e){for(var n=1;n<arguments.length;n++){var t=null!=arguments[n]?arguments[n]:{};n%2?o(Object(t),!0).forEach((function(n){a(e,n,t[n])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(t)):o(Object(t)).forEach((function(n){Object.defineProperty(e,n,Object.getOwnPropertyDescriptor(t,n))}))}return e}function s(e,n){if(null==e)return{};var t,i,a=function(e,n){if(null==e)return{};var t,i,a={},o=Object.keys(e);for(i=0;i<o.length;i++)t=o[i],n.indexOf(t)>=0||(a[t]=e[t]);return a}(e,n);if(Object.getOwnPropertySymbols){var o=Object.getOwnPropertySymbols(e);for(i=0;i<o.length;i++)t=o[i],n.indexOf(t)>=0||Object.prototype.propertyIsEnumerable.call(e,t)&&(a[t]=e[t])}return a}var l=i.createContext({}),c=function(e){var n=i.useContext(l),t=n;return e&&(t="function"==typeof e?e(n):r(r({},n),e)),t},u=function(e){var n=c(e.components);return i.createElement(l.Provider,{value:n},e.children)},p={inlineCode:"code",wrapper:function(e){var n=e.children;return i.createElement(i.Fragment,{},n)}},d=i.forwardRef((function(e,n){var t=e.components,a=e.mdxType,o=e.originalType,l=e.parentName,u=s(e,["components","mdxType","originalType","parentName"]),d=c(t),m=a,h=d["".concat(l,".").concat(m)]||d[m]||p[m]||o;return t?i.createElement(h,r(r({ref:n},u),{},{components:t})):i.createElement(h,r({ref:n},u))}));function m(e,n){var t=arguments,a=n&&n.mdxType;if("string"==typeof e||a){var o=t.length,r=new Array(o);r[0]=d;var s={};for(var l in n)hasOwnProperty.call(n,l)&&(s[l]=n[l]);s.originalType=e,s.mdxType="string"==typeof e?e:a,r[1]=s;for(var c=2;c<o;c++)r[c]=t[c];return i.createElement.apply(null,r)}return i.createElement.apply(null,t)}d.displayName="MDXCreateElement"},8827:function(e,n,t){t.r(n),t.d(n,{assets:function(){return u},contentTitle:function(){return l},default:function(){return m},frontMatter:function(){return s},metadata:function(){return c},toc:function(){return p}});var i=t(7462),a=t(3366),o=(t(7294),t(3905)),r=["components"],s={title:"Single Nucleotide Variation",track:"Genomic Variation"},l=void 0,c={unversionedId:"genomic-variation/single-nucleotide-variation",id:"genomic-variation/single-nucleotide-variation",title:"Single Nucleotide Variation",description:"[Single Nucleotide Variants (SNVs) and Single Nucleotide Polymorphisms",source:"@site/docs/02-genomic-variation/02-single-nucleotide-variation.md",sourceDirName:"02-genomic-variation",slug:"/genomic-variation/single-nucleotide-variation",permalink:"/docs/genomic-variation/single-nucleotide-variation",draft:!1,editUrl:"https://github.com/stjude/learngenomics.dev/tree/main/docs/02-genomic-variation/02-single-nucleotide-variation.md",tags:[],version:"current",sidebarPosition:2,frontMatter:{title:"Single Nucleotide Variation",track:"Genomic Variation"},sidebar:"tutorialSidebar",previous:{title:"Baselines for Variation",permalink:"/docs/genomic-variation/baselines-for-variation"},next:{title:"Insertions and Deletions",permalink:"/docs/genomic-variation/insertions-and-deletions"}},u={},p=[],d={toc:p};function m(e){var n=e.components,s=(0,a.Z)(e,r);return(0,o.kt)("wrapper",(0,i.Z)({},d,s,{components:n,mdxType:"MDXLayout"}),(0,o.kt)("p",null,(0,o.kt)("a",{parentName:"p",href:"https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism"},"Single Nucleotide Variants (SNVs) and Single Nucleotide Polymorphisms\n(SNPs)"),"\xa0are the\nsmallest unit of variation in the genome. Here, just one nucleotide, or base, is\nsubstituted at a single position in the genome (such as changing a 'C' to an 'A'\non chromosome 5, position 1,485,234). The term\xa0\"",(0,o.kt)("strong",{parentName:"p"},'SNV"'),"\xa0is used broadly to\ndescribe an event where a single nucleotide substitution, insertion or deletion\noccurs.\xa0Single nucleotide polymorphisms, or ",(0,o.kt)("strong",{parentName:"p"},"SNPs"),",\xa0are a subset of SNVs where\nthe change is found in some non-trivial percentage of a population (e.g. ",">"," 1%,\nthough an exact threshold is not universally agreed upon). By contrast, SNVs\nthat are not considered SNPs can be very rare and may cause cancer or other\ndiseases."),(0,o.kt)("p",null,(0,o.kt)("img",{alt:"An example single nucleotide variation where an &#39;A&#39; is substituted in for a &#39;C&#39;",src:t(7631).Z,width:"2439",height:"664"})),(0,o.kt)("p",null,'In cancer genomics, SNPs are less interesting to researchers than SNVs due to\ntheir prevalence within the population. To illustrate, consider a SNP event\nwhere 45% of the people have an "A" at the specified position and 55% have a "T"\nat that position. 45% of the human population does not have cancer, so the\nvariation is less likely to play a major role in oncogenesis (though they can\nabsolutely play a role in cancer predisposition).\nThe\xa0',(0,o.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/snp/"},"dbSNP"),"\xa0database is perhaps the most\nwidely-used population frequency database."),(0,o.kt)("p",null,"Determining which of the hundreds of thousands of possible SNVs are important in\na particular disease is done by statistical association or through applying\nbiological knowledge. Genome-wide association studies (",(0,o.kt)("strong",{parentName:"p"},"GWAS"),") collect data\nfrom thousands of patient samples with the same disease and statistically\ncompare their SNVs to those in a normal population. GWAS studies work well for\ncommon diseases that are well-defined and reliably diagnosed. This is especially\ntrue if the number of disease-causing variants is small. However, most diseases\nhave many genetic variants that influence the disease. Importantly, GWAS studies\nonly find variants that are linked to disease more closely than expected by\nchance. Very often these variants only indicate an interesting genomic location\nand do not themselves result in disease."),(0,o.kt)("p",null,"One downside of GWAS studies is that they require a relatively large number of\nsamples to gain enough statistical power. In rare diseases like pediatric\ncancer, GWAS studies are not as practical due to low sample count. Thus,\nresearchers apply biological knowledge ",(0,o.kt)("em",{parentName:"p"},"a priori")," to improve the odds of finding\nrelevant SNVs. Commonly scientists focus their efforts on SNVs that fall within\nexons of genes that are known to be associated with cancer or that are predicted to\nalter the protein product. Databases such\nas\xa0",(0,o.kt)("a",{parentName:"p",href:"https://cancer.sanger.ac.uk/cosmic"},"COSMIC"),"\xa0track variants in cancer genomes\nto help identify genes and genomic locations that are commonly mutated and are,\ntherefore, more likely to be relevant to disease."),(0,o.kt)("p",null,'As an aside, SNVs and SNPs are often both colloquially referred to "snips".\nStrictly speaking, SNVs are not all SNPs, but it is difficult to imagine an easy\npronunciation for "SNVs", so individuals often mix the two concepts in speech\n(e.g. "we need to look at the called snips"). You\'ll need to use context to\ndelineate whether one is talking about polymorphisms or just single nucleotide\nvariation in general.'))}m.isMDXComponent=!0},7631:function(e,n,t){n.Z=t.p+"assets/images/2.1-SNV-0619c3e681f3bb21a1bb5ea0e5f401e4.jpg"}}]);